src/test/java/org/elasticsearch/index/analysis/TestIntegration.java

```diff
@@ -84,7 +92,7 @@ public void testAnalysis() throws IOException {
                 new int[]{ 0,  0,  0,  5,  7,  7, 14, 14},
                 new int[]{ 4,  4, 18,  6, 13, 13, 18, 18},
                 null,
-                new int[]{ 1,  0,  0,  1,  1,  0,  1,  0});
+                new int[]{ 0,  0,  0,  1,  2,  2,  3,  3});
```

Comment:
Changed this to mean the actual positions of the tokens, instead of their increment. This is what we are testing after all. Also, the test broke due to ES analysis API changes.